alacritty_terminal/src/config/window.rs

```diff
@@ -69,6 +69,24 @@ impl WindowConfig {
     }
 }
 
+impl Default for WindowConfig {
+    fn default() -> WindowConfig {
+        WindowConfig {
+            dimensions: Default::default(),
+            position: Default::default(),
+            padding: Default::default(),
+            decorations: Default::default(),
+            dynamic_padding: Default::default(),
+            startup_mode: Default::default(),
+            title: DEFAULT_NAME.into(),
```

Comment:
We should probably call the `default_title` function here for consistency.

Reply:
Yeah, I was going back and forth on that one. I'll change it.